### DISCUSSION TITLE
PrimeNG License Model Question

### DISCUSSION THREAD

**Discussion by demetrius-powers** [primefaces/community#1721] [PrimeNG] [answered]
Hi,

If I build an angular app using PrimeNG.  

Install it 10 customer locations will i need to purchase a license to use for each customer as I have purchased the basic one as of today

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

PrimeNG is an open source library it's okay to do it. However, if you're an LTS, PrimeBlocks, or any Prime UI Template user,  which are not open source, please send email to contact@primetek.com.tr for more information.
